lib/node_modules/@stdlib/stats/base/dists/arcsine/README.md

```diff
@@ -107,10 +107,36 @@ var mu = dist.mean;
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+// Display Available Functions
 var objectKeys = require( '@stdlib/utils/keys' );
 var arcsine = require( '@stdlib/stats/base/dists/arcsine' );
-
 console.log( objectKeys( arcsine ) );
+
+// Calculate the Median for the arcsine distribution in the range [0, 1]
+var medianValue = arcsine.median(0, 1);
+console.log('The median of the distribution is', medianValue);
+
+// Calculate the Mode for the arcsine distribution in the range [0, 1]
+var modeValue = arcsine.mode(0, 1);
+console.log('The mode of the distribution is', modeValue);
+
+// Calculate the Entropy for the arcsine distribution in the range [0, 1]
+var log = require( '@stdlib/math/base/special/log' );
```

Comment:
Why is this here?

Reply:
This was a requirement in the issue description

> [Examples may require functionality from other packages to help make examples more interesting.](https://github.com/stdlib-js/stdlib/issues/1612)

Reply:
The import must make sense. As is, you don't actually use `log` anywhere in the examples.

Reply:
understood, i have fixed it